### PR TITLE
Change opencv version from 2.7 to 3.2

### PR DESCRIPTION
**Pull request by chutsu**
PR to change libwave opencv version from 2.7 to 3.2.

- Modify opencv3 install script to pull from github
- Update `install_deps.bash` to install opencv3

### CODE REVIEW

File: scripts/install/opencv3_install.bash [hunk 1]
```diff
@@ -41,20 +41,17 @@ install_dependencies() {
...
 download_opencv() {
-    mkdir -p /usr/local/src/opencv
-    cd /usr/local/src/opencv
-    if [ ! -d opencv-3.0.0-alpha ]; then
-        wget $OPENCV_URL -O opencv-3.0.0-alpha.zip
-        unzip -qq opencv-3.0.0-alpha.zip
-        rm opencv-3.0.0-alpha.zip
+    cd /usr/local/src/
+    if [ ! -d opencv ]; then
+        git clone $OPENCV_REPO
```
Comment (leokoppel) [inline scripts/install/opencv3_install.bash]: Downloading the whole repo might slow down CI. Can we still download a zip?

Comment (chutsu) [inline scripts/install/opencv3_install.bash]: good point yes.

Comment (chutsu) [inline scripts/install/opencv3_install.bash]: Good point, yes.

File: scripts/install/opencv3_install.bash [hunk 2]
```diff
@@ -43,18 +43,18 @@ install_dependencies() {
 download_opencv() {
     mkdir -p /usr/local/src/opencv
     cd /usr/local/src/opencv
-    if [ ! -d opencv-3.0.0-alpha ]; then
-        wget $OPENCV_URL -O opencv-3.0.0-alpha.zip
-        unzip -qq opencv-3.0.0-alpha.zip
-        rm opencv-3.0.0-alpha.zip
+    if [ ! -d opencv-3.2.0 ]; then
+        wget $OPENCV_URL -O opencv3.2.zip
```
Comment (navganti) [inline scripts/install/opencv3_install.bash]: Up above it says 3.2.0, here it says 3.2. Is that a typo?

Comment (chutsu) [inline scripts/install/opencv3_install.bash]: oh that's just renaming the file to 3.2

Comment (navganti) [inline scripts/install/opencv3_install.bash]: Oh whoops, my bad. Not a big deal, was just curious.

Comment (chutsu) [inline scripts/install/opencv3_install.bash]: too late fixed it

**Pull request merged**